java-analyzer-bundle.core/src/main/java/io/konveyor/tackle/core/internal/symbol/SymbolProviderResolver.java

```diff
@@ -17,9 +19,10 @@ public class SymbolProviderResolver {
         map.put(7, new ReturnTypeSymbolProvider());
         map.put(8, new ImportSymbolProvider());
         map.put(9, new VariableDeclarationSymbolProvider());
+        map.put(10, new VariableDeclarationSymbolProvider());
```

Comment:
9, 10 are same, is that intentional? I see above 10 is TypeSymbol

Reply:
+1 need to fix that